dmriqcpy/viz/screenshot.py

```diff
@@ -494,19 +495,19 @@ def screenshot_tracking(tracking, t1, directory="."):
 
     streamlines = []
     it = 0
-    for streamline in tractogram:
+    for streamline in sft.streamlines:
```

Comment:
can you make it consistent? I like streamline in sft.streamlines better than stream